Taskfile.yml

```diff
@@ -0,0 +1,90 @@
+version: '3'
+
+tasks:
+  install:
+    cmds:
+      - echo "Installing dependencies"
+      - cmd: sudo apt install -y libgtk-3-dev libwebkit2gtk-4.0-dev
+        platforms: [linux]
+      - cmd: go install github.com/go-swagger/go-swagger/cmd/swagger@latest
+      - cmd: go install github.com/wailsapp/wails/v2/cmd/wails@latest
+
+  generate: go generate ./...
+
+  test: go test -v ./...
+
+  fmt:
+    cmds:
+      - cmd: gofumpt -w .
+      - cmd: golangci-lint run
+
+  build:
+    cmds:
+      - task: build-linux
+      - task: build-macos
+      - task: build-windows
+
+  build-linux:
+    platforms: [linux]
+    cmds:
+      - cmd: mkdir -p build/wallet-plugin
+      - cmd: go build -o build/wallet-plugin/wallet-plugin -tags desktop,production  -ldflags "-w -s" main.go
+
+  build-macos:
+    platforms: [darwin]
+    cmds:
+      - cmd: mkdir -p build/wallet-plugin
+      - cmd: go build -tags desktop,production -ldflags "-w -s" -o build/wallet-plugin/wallet-plugin main.go
+    env:
+      CGO_LDFLAGS: -framework UniformTypeIdentifiers
+
+  build-windows:
+    platforms: [windows]
+    cmds:
+      - cmd: go build -tags desktop,production -ldflags "-w -s -H windowsgui" -o build/wallet-plugin/wallet-plugin.exe main.go
```

Comment:
it will automatically create build/wallet-plugin folder?

Reply:
good catch! i added mkdir and changed the slashes side.. but its not tested :)